verde/base/base_classes.py

```diff
@@ -451,22 +466,23 @@ def profile(
         return pd.DataFrame(dict(columns), columns=[c[0] for c in columns])
 
 
-def get_dims(dims):
+def get_dims(gridder, dims):
```

Comment:
Might be better off as a private method in this case.

Reply:
Done!